docs/projects/index.md

```diff
@@ -4,24 +4,23 @@ The regular disclosure of structured data can greatly enhance the transparency a
 
 ## What is a project?
 
-An infrastructure project may stand alone (e.g. a town building a new hospital), or may form part of a wider programme of activity (e.g. construction of a number of railway stations). Each infrastructure project will usually involve project-level planning and co-ordination, managed through a series of contracts and subcontracts. 
+An infrastructure project may stand alone (e.g. a new hospital), or may form part of a wider programme of work (e.g. a new rail station, as part of an extension to a railway line). An infrastructure project is usually implemented through a series of contracts, and usually involves planning and coordination at the project-level.
 
 ```eval_rst
 .. admonition:: Tip
     :class: tip
 
     .. markdown::
 
-      The term 'project' can also be used to refer to a larger programme of work of which an individual infrastructure project is a part, such as the construction of a whole highway. The term 'project' may also refer to a public investment project with funds allocated in the national budget, such as the upgrading of the rail network in a particular region.
-
-      However, in OC4IDS, the term 'project' refers to the construction of a single asset, for example the construction of a bridge, a highway junction or building.
+      In the context of OC4IDS, the term 'project' is defined as the delivery of a set of infrastructure assets in a specified location, generally under a single managing entity (project owner) and budget authority: for example, the construction of a bridge, highway junction or building.
```

Comment:
To which terms do "single managing entity (project owner) and budget authority" relate within the project schema? If there is a correspondence, it would be clearer to reuse the same terms.

I don't like that guidance note 6 defines it as "the delivery of", as we know a project is not one phase… In the documentation, we can abbreviate to "the term 'project' refers to a set of …"

Given the new definition, we should update the examples, which are all single assets. We can update it to a single asset and a set of assets (perhaps "; for example, a highway overpass or a university campus").

Reply:
Discussed "a single procuring entity and ".

Reply:
I've made these updates in https://github.com/open-contracting/infrastructure/pull/106/commits/bcef0830fc95a01edd2c4e49c8e53a42e7a6e5ee, I also replaced 'under' with the 'the responsibility of" 

However, on reflection I'm not sure removing 'delivery of' actually makes the description clearer for the reader, since the new definition of 'a set of infrastructure assets in a specified location' could refer to an already existing set of assets for which no further work is planned (rather than a project to create new assets or update existing assets).

Since it would be over long to list all the project phases (identification, preparation, implementation and completion) we could instead use a catch all term which isn't used elsewhere in the schema or documentation (unlike delivery), e.g.

> In the context of OC4IDS, the term 'project' refers to the development of a set of infrastructure assets in a specified location, generally the responsibility of a single procuring entity and budget authority: for example, a highway overpass or a university campus.

Reply:
That works for me. Feel free to make the change in the two locations and merge.